mass2/core/channel.py

```diff
@@ -373,9 +373,14 @@ def typical_peak_ind(self, col="pulse"):
     def summarize_pulses(self, col="pulse", pretrigger_ignore_samples=0, peak_index=None) -> "Channel":
         if peak_index is None:
             peak_index = self.typical_peak_ind(col)
+        out_names = mass2.core.pulse_algorithms.result_dtype.names
+        if out_names is None:
```

Comment:
this seems like an uneccesary if, we should just error if its None, nothing else will work

Reply:
I used an assertion and explained it in a comment. Fixed in c5c0631c.